x-pack/plugins/index_management/public/application/lib/index_mode_labels.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const getIndexModeLabel = (mode?: string | null) => {
+  switch (mode) {
+    case 'standard':
+    case null:
```

Comment:
Can these empty cases be removed?

Reply:
`null` is a valid value accepted by Es (is is actually the default value) and it is same as `standard`. Included `undefined` in this switch case as well because it means the index mode is not specified and so it defaults to `null`. We still need these cases in this function as it is used in the Index template creation flow summary, where the user could have typed `null` or not specified the index mode setting at all.

Reply:
Ah, its been a while since I've seen the fall through feature of switch statements used and therefore misread the code.